src/test/consensus/Consensus_test.cpp

```diff
@@ -93,6 +94,7 @@ class Consensus_test : public beast::unit_test::suite
         using namespace csf;
         using namespace std::chrono;
 
+        testcase("slow peer");
```

Comment:
There are fewer hard rules in test code. I use test cases sparingly, to give people confidence that the test is comprehensive. But not so many that it looks like noise. Its up to you to decide the utility of the test cases.